mne/viz/topomap.py

```diff
@@ -1085,7 +1085,8 @@ def plot_evoked_topomap(evoked, times=None, ch_type='mag', layout=None,
                               mask=mask_[:, i] if mask is not None else None,
                               mask_params=mask_params, axis=ax,
                               outlines=outlines, image_mask=image_mask,
-                              contours=contours, image_interp=image_interp)
+                              contours=contours, image_interp=image_interp,
+                              show=show)
```

Comment:
`show=False` so the showing gets delayed until the end of the function call on line 1125